core/src/program/mod.rs

```diff
@@ -43,10 +52,47 @@ impl Program {
     pub fn hash(&self) -> Digest {
         self.root.hash()
     }
+
+    /// Returns code block table for this program.
+    pub fn cb_table(&self) -> &CodeBlockTable {
+        &self.cb_table
+    }
 }
 
 impl fmt::Display for Program {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         write!(f, "begin {} end", self.root)
     }
 }
+
+// CODE BLOCK TABLE
+// ================================================================================================
+
+/// A map of code block hashes to their underlying code blocks.
+///
+/// This table is used to hold code blocks which a referenced from the program MAST but are
```

Comment:
nit: this still has a typo (there were 2 before). It says "a referenced" instead of "are referenced"

Reply:
Ah yes - sorry! fixed!